.eslintrc.json

```diff
@@ -38,7 +42,10 @@
       }
     ],
     "class-methods-use-this": "error",
-    "comma-dangle": "error",
+    "comma-dangle": [
+      "error",
+      "always-multiline"
```

Comment:
uniformity across all, only-multiline will allow for dangling-comma to be absent, something we dont want..
again discussion about it in the comments https://github.com/ManageIQ/manageiq/issues/8781
@himdel can probably articulate the argument for a bit more eloquently than myself :)